system-addon/lib/TopSitesFeed.jsm

```diff
@@ -76,7 +76,7 @@ this.TopSitesFeed = class TopSitesFeed {
     const newAction = {type: at.TOP_SITES_UPDATED, data: links};
 
     // Send an update to content so the preloaded tab can get the updated content
-    this.store.dispatch(ac.SendToContent(newAction, action.meta.fromTarget));
+    this.store.dispatch(ac.BroadcastToContent(newAction));
```

Comment:
This isn't quite right as we do want to send just to the single content target on NEW_TAB_LOAD but broadcast for history clear. I suppose we could probably check if action.meta.fromTarget exists and assumes if there is none, it's meant to broadcast?